profiler/src/ProfilerEngine/Datadog.Profiler.Native/CorProfilerCallback.cpp

```diff
@@ -619,7 +619,7 @@ bool CorProfilerCallback::StopServices()
         }
         else
         {
-            Log::Error(name, " failed to stop.");
+            Log::Info(name, " failed to stop.");
```

Comment:
Why not keeping an error here?
This is what is done for StartServices()

Reply:
Otherwise it fails the tests.
Actually, we stop some services in other places without taking into account if it was successful or not so....